lib/rules/quotes.js

```diff
@@ -14,6 +14,18 @@ export default class Quotes extends Rule {
       }
       case 'string': {
         if (['double', 'single'].includes(config)) {
+          return {
+            hbs: config,
+            html: config,
+          };
+        }
+        break;
+      }
+      case 'object': {
+        if (
+          ['double', 'single'].includes(config.hbs) &&
+          ['double', 'single'].includes(config.html)
```

Comment:
Wouldn't it make sense to allow specifying one of them, and falling back to the default for the one not specified? And note we can't change defaults now (only in a future breaking change PR).

Reply:
The problem with this is the default is actually off. The "double" is part of the stylistic preset. I think it's best to just disable the rule for the one not specified, much like your other suggestion of when it is false.